.gitignore

```diff
@@ -1,5 +1,6 @@
 /.project
+/.idea
 /release
 /jiralert
 /jiralert*.tar.gz
-
+/vendor
```

Comment:
Do we still need the `vendor` directory, after this change? Why not just drop it?

Reply:
That's why I am adding this (: To ignore this for sure. 

Essentially there is a command for go mod to generate it. I am just making sure it won't be checked in. (:

Reply:
Shouldn't you also delete the existing contents of `vendor` from the repository?

Reply:
... :man_facepalming:  I missed there is a one!